### PR TITLE
Handle also paired variablesList

### PR DESCRIPTION
**Pull request by boutinb**
The Bain Paired T-Tests is in preloadData mode, and should work.

To test, load the bainPairedSamples JASP file in the Data Library/Bain folder, and refresh it.
Test also Paired T-Tests of the TTests module.

Fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2678

### CODE REVIEW

**Review by koenderks — APPROVED** [review]
Seems to me like it is working correctly

File: QMLComponents/boundcontrols/boundcontrolmultiterms.cpp [hunk 1]
```diff
@@ -71,3 +73,36 @@ void BoundControlMultiTerms::resetBoundValue()
...
+{
+	Json::Value newValue;
+
+	if (_control->encodeValue())
+	{
+		if (_isValueWithTypes(value))
+			newValue = value;
+		else
+		{
+			Json::Value types(Json::arrayValue);
```
Comment (JorisGoosen) [inline QMLComponents/boundcontrols/boundcontrolmultiterms.cpp:88]: So this "else" is *just* for loading pre-0.19.2 versions no?

Comment (JorisGoosen) [reply QMLComponents/boundcontrols/boundcontrolmultiterms.cpp:88]: If so maybe add a comment here a la 
```suggestion
			// Else we are loading from a jasp version before "preloadData" was on or var.types were added to the options
			Json::Value types(Json::arrayValue);
```

**Review by JorisGoosen — APPROVED** [review]

**Comment by JorisGoosen** [comment]
It all works fine and the code seems good but I think maybe a comment might be warranted in the code where I mention this ;)

**Pull request merged**